integration/ztoc_test.go

```diff
@@ -145,3 +179,248 @@ func ztocExistChecker(t *testing.T, listOutputLines []string, img testImageIndex
 	t.Fatalf("invalid ztoc from index %s for image %s:\n expected ztoc: digest: %s, size: %s, layer digest: %s\n actual output lines: %s",
 		img.sociIndexDigest, img.imgInfo.ref, ztocDigest, size, layerDigest, listOutputLines)
 }
+func TestSociZtocInfo(t *testing.T) {
+	t.Parallel()
+	sh, done := newSnapshotterBaseShell(t)
+	defer done()
+	rebootContainerd(t, sh, "", "")
+
+	testImages := prepareSociIndices(t, sh)
+
+	getFullZtoc := func(sh *dockershell.Shell, ztocPath string) (*ztoc.Ztoc, error) {
+		output := sh.O("cat", ztocPath)
+		reader := bytes.NewReader(output)
+		z, err := ztoc.Unmarshal(reader)
+		return z, err
+	}
+
+	verifyInfoOutput := func(t *testing.T, zinfo Info, ztoc *ztoc.Ztoc) error {
```

Comment:
`t *testing.T` is not used in the `func`, can you remove it? Also can you move `verifyInfoOutput` to a separate func (put it in this file is fine) since it's long and not use other variables outside of it.